### DISCUSSION TITLE
关于运行run.py   执行 报错  No predictions found.

### DISCUSSION THREAD

**Discussion by GOOD-N-LCM** [open-compass/opencompass#1297] [Q&A]
![image](https://github.com/open-compass/opencompass/assets/69232008/9fa53707-73d6-49bd-ab34-e6817aaa89b5)

在其他网站教程 提示  安装lmdeploy。  pip  install  lmdeploy 之后依旧是这个错误。求大佬帮解答

**Comment by liuchunming033** [comment]
遇到了同样的问题，请问是如何解决的？

**Comment by liuwake** [comment]
Check this out, [我解决了torchrun环境异常情况下的No predictions found问题. ](https://github.com/open-compass/opencompass/discussions/1370#discussion-6984033)
